src/conformance/build.gradle

```diff
@@ -42,7 +42,6 @@ android {
         externalNativeBuild {
             cmake {
                 arguments '-DANDROID_STL=c++_shared',
-                        '-DBUILD_API_LAYERS=ON',
```

Comment:
Was there a particular API layer that didn't build right? They should all build, though there's no special Android support for loading them so you have to have writable root and install them as if they were linux. The conformance test does require the use of the conformance layer, which currently is needed this way.

Reply:
When `BUILD_API_LAYERS` is enabled, there will be a building error:

```
> ~/openxr/OpenXR-CTS/CMakeLists.txt : C/C++ debug|armeabi-v7a : CMake Error at ~/openxr/OpenXR-CTS/src/CMakeLists.txt:424 (add_subdirectory):
    add_subdirectory given source "api_layers" which is not an existing
    directory.
```

Reply:
And there is a logic to enable `BUILD_API_LAYERS` automatically at `src/CMakeLists.txt` when `api_layers/CMakeLists.txt` existed:

```
if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/api_layers/CMakeLists.txt")
    option(BUILD_API_LAYERS "Build API layers" ON) 
endif()
```

Reply:
ah, great, good catch then.